containers/Balancer.ts

```diff
@@ -6,6 +6,7 @@ import { TOKENS, POOL } from "../apollo/balancer/queries";
 
 import Connection from "./Connection";
 import Token from "./Token";
+import { YIELD_TOKENS } from "../utils/yieldTokenList";
```

Comment:
I moved this YIELD_TOKENS struct outside the Balancer container because I didn't want to force components to load all of the balancer data just to check whether a token address was a yield token